index.html

```diff
@@ -73,7 +77,7 @@
 
   <div id="map"></div>
 
-  <script data-spreadsheet="0Agcr__L1I1PDdEpoMnhxR0RHdkFsWlFtNTlEZlltR0E" src="js/script.js"></script>
+  <script data-spreadsheet="0Agcr__L1I1PDdEpoMnhxR0RHdkFsWlFtNTlEZlltR0E" data-actionkit="peoples-climate-plan-actions|break-free-world" src="js/script.js"></script>

```

Comment:
Actually, is there any reason not to just put this in `data-map-args`?

https://github.com/350dotorg/350map.org/blob/gh-pages/js/script.js#L3

``
<script data-spreadsheet="0Agcr__L1I1PDdEpoMnhxR0RHdkFsWlFtNTlEZlltR0E" data-map-args="actionkit=peoples-climate-plan-actions|break-free-world" src="js/script.js"></script>
``

That would also take care of #5 (allowing the integrator to configure this in the iframe url)